src/context/EarnManagerProvider/providers/yearn/components/YearnManager/YearnDeposit.tsx

```diff
@@ -159,14 +292,19 @@ export const YearnDeposit = ({ api }: YearnDepositProps) => {
         return (
           <Approve
             asset={asset}
-            cryptoEstimatedGasFee=''
+            feeAsset={feeAsset}
+            cryptoEstimatedGasFee={bnOrZero(state.approve.estimatedGasCrypto)
+              .div(`1e+${feeAsset.precision}`)
```

Comment:
do we need to check if `feeAsset.precision` is defined here to avoid a `NaN` value?

Reply:
No these are required fields on the asset